recipes/benchmark/all/conanfile.py

```diff
@@ -52,6 +49,10 @@ def validate(self):
         if Version(self.version) < "1.7.0" and is_msvc(self) and self.info.options.shared:
             raise ConanInvalidConfiguration(f"{self.ref} doesn't support msvc shared builds")
 
+    # TODO: it is required for several CI environment(ex. linux gcc 11)
+    def build_requirements(self):
+        self.tool_requires("cmake/3.16.3")
```

Comment:
c3i worked fine in previous PR, why is it required now? Seems excessive.

Reply:
@SpaceIm 
Sorry, I missed.
I'm going to revert it.

Reply:
well, it is needed for benchmark 1.7.1:

```
benchmark/1.7.1: 
CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
  CMake 3.16.3 or higher is required.  You are running version 3.15.7
```

Reply:
@SpaceIm  @ericLemanissier 
I think it would be misleading to `build_require` cmake for CI.
As an alternative, I created a patch to downgrade `cmake_minimum_required`.
This seems to build without any problem.

Reply:
I don't understand what is misleading. There are other recipes tool/build_requiring cmake. This happens every time the upstream project needs a newer version than what C3I provides. This seems to me highly preferable than to create a patch.

Reply:
@ericLemanissier 
I understand.
Let's revert it.